azure-pipelines.yml

```diff
@@ -36,6 +36,7 @@ steps:
 - script: |
     pushd '$(projectRoot)'
     python runner.py "Tests/TestConfiguration.json" "$(PS_BATCH_ACCOUNT_NAME)" "$(BatchAccountKey)" "$(PS_BATCH_ACCOUNT_URL)" "$(PS_BATCH_ACCOUNT_SUB)" "$(PS_STORAGE_ACCOUNT_NAME)" "$(StorageAccountKey)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_CLIENT_ID)" "$(ServicePrincipalCredentialsSecret)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_TENANT)" "$(PS_SERVICE_PRINCIPAL_CREDENTIALS_RESOUCE)" 
+    cat template.log
```

Comment:
is this just for debugging or should it stay here?

Reply:
If we don't have cat we won't be able to see the logs in the 'Run Tests' task